src/app.js

```diff
@@ -1 +1,50 @@
-console.log('Hallo daar!');
\ No newline at end of file
+//Import axios
+import axios from "axios";
+
+//Maak een async functie waarbij de benodigde informatie wordt opgehaald
+
+async function getCountryDataList() {
+
+    try {
+        const result = await axios.get('https://restcountries.com/v2/all?fields=name,region,flags,population');
+        console.log(result.data);
+
+        result.data.sort((a, b) =>{
+            return a.population - b.population;
+        })
+
+        getAllCountries(result.data);
+
+    } catch (error) {
+        console.error(error);
+    }
+}
+getCountryDataList();
+
+function getAllCountries(countries) {
+    const countryUnorderedList = document.getElementById('country-list');
+
+    countries.map((allCountries) => {
+
+        const countryList = document.createElement('li');
+        countryList.innerHTML = `
+            <img class="flag" src="${allCountries.flags.png}" alt="flag">
+            <h3 class="${allCountries.region}">${allCountries.name}</h3>
```

Comment:
Door allCountries.region vallen noord en zuid america onder dezelfde class. Als je hier nog onderscheid in wilt maken kan je een nieuwe functie maken dat de juiste class naam geeft door gebruikt te maken van de region en de subRegion.